README.md

```diff
@@ -7,6 +7,11 @@ An Ansible role for installing [Uncomplicated
 Firewall](https://wiki.ubuntu.com/UncomplicatedFirewall?action=show&redirect=UbuntuFirewall)
 (UFW).
 
+> [!NOTE]
+> We cannot currently test *any* ARM64 platforms under `qemu` because
+> [`qemu` cannot currently support
```

Comment:
Consistent capitalization of QEMU:
```suggestion
> We cannot currently test *any* ARM64 platforms under QEMU because
> [QEMU cannot currently support
```